oak-it-osgi/src/test/java/org/apache/jackrabbit/oak/osgi/TikaExtractionOsgiIT.java

```diff
@@ -116,15 +116,16 @@ private Option setupTikaAndPoi() throws IOException {
                         , wrappedBundle(mavenBundle("org.apache.poi", "poi", versions.get(POI_VERSION)))
                         , wrappedBundle(mavenBundle("org.apache.poi", "poi-scratchpad", versions.get(POI_VERSION)))
                         , wrappedBundle(mavenBundle("org.apache.poi", "poi-ooxml", versions.get(POI_VERSION)))
-                        , wrappedBundle(mavenBundle("org.apache.poi", "poi-ooxml-schemas", versions.get(POI_VERSION)))
+                        , wrappedBundle(mavenBundle("org.apache.poi", "poi-ooxml-lite", versions.get(POI_VERSION)))
                                 .instructions("DynamicImport-Package=*")
                         , wrappedBundle(mavenBundle("org.apache.poi", "ooxml-security", "1.0"))
-                        , wrappedBundle(mavenBundle("org.apache.xmlbeans", "xmlbeans", "3.1.0"))
+                        , wrappedBundle(mavenBundle("org.apache.xmlbeans", "xmlbeans", "5.0.3"))
                         , wrappedBundle(mavenBundle("com.drewnoakes", "metadata-extractor", "2.6.2"))
                         , mavenBundle("org.apache.commons", "commons-collections4", versions.get(COLLECTIONS4_VERSION))
                         , mavenBundle("org.apache.commons", "commons-compress", versions.get(COMPRESS_VERSION))
                         , mavenBundle("org.apache.commons", "commons-lang3", versions.get(LANG3_VERSION))
                         , mavenBundle("org.apache.commons", "commons-math3", versions.get(MATH3_VERSION))
+                        , mavenBundle("org.apache.logging.log4j", "log4j-api", "2.17.2")
```

Comment:
Could you please document why we need this change (I suspect we need it, but it would be good to have a record on why).

Reply:
@reschke The reason was, we moved from poi 4.x to poi 5.x and log4j-api is a dependency for poi 5.x.

But now instead of log4j-api, I upgraded pax version to handle this requirement.